source/features/useful-not-found-page.tsx

```diff
@@ -108,6 +109,15 @@ function init(): false | void {
 	}
 }
 
+async function initPRCommit(): Promise<void | false> {
+	const commitUrl = location.pathname.replace(/\/$/, '').replace(/\/pull\/\d+\/commits/, '/commit');
```

Comment:
I think the "strip slash" can be avoided and the replacement can be slightly simplified
```suggestion
	const commitUrl = location.pathname.replace(/pull\/\d+\/commits/, 'commit');
```